### ISSUE TITLE
Report Bug in cross-model transfer

### ISSUE THREAD

**Issue by Celine-hxy** [thunlp/Prompt-Transferability#2]
crossPrompt.py   line317
**mask_logits = logits[:, 0]** should be modified to **mask_logits = logits[:, 100]** because there are 100 prompt tokens in the front.

**Comment by Celine-hxy** [comment]
crossPromptT5.py  line 96~98: 
**self.encoder.prompt_embeddings.weight.data = target_prompt
self.encoder.encoder.prompt_tokens.weight.data = target_prompt
self.encoder.decoder.prompt_tokens.weight.data = target_prompt**
Assigning weights in this way will result in grad_ fn lost. Hence gradient cannot be backpropagated.

**Comment by yushengsu-thu** [comment]
@Celine-hxy these bugs are already fixed.
